ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,23 @@
 <div class="container-fluid wu-frame">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
The styles here look nothing like the styles in the rest of the app and also do not appear to look like the OpenShift styles. Just use the hasError mechanism and the builtin styles.